ci-operator/step-registry/telcov10n/functional/cnf-network/ocp-operators/telcov10n-functional-cnf-network-ocp-operators-ref.yaml

```diff
@@ -0,0 +1,31 @@
+ref:
+  as: telcov10n-functional-cnf-network-ocp-operators
+  from_image:
+    namespace: telcov10n-ci
+    name: eco-ci-cd
+    tag: eco-ci-cd
+  commands: telcov10n-functional-cnf-network-ocp-operators-commands.sh
+  timeout: 4h
+  resources:
+    requests:
+      cpu: 100m
+  env:
+  - name: VERSION 
+    default: "4.17"
+    documentation: OCP version to deploy
```

Comment:
Isn't this the `channel`? Not all of the operators use OCP versions for the channel

Reply:
No channel can be specified by different argument. This arguments is in use for stage an later will be used for brew as well.